### ISSUE TITLE
Bar displays junk characters

### ISSUE THREAD

**Issue by oliverrausch** [jaagr/polybar#196]
My polybar config is displaying seemingly random characters after the volume and backlight modules.
[image](http://imgur.com/a/Bgq5n)

This is my polybar config

```
[global/wm]
margin-top = 5
margin-bottom = 5

[colors]
background	=#ee222222
foreground	=#dfdfdf
backlight	=#d8ad4c 
volume 		=#cf6a4c 
memory		=#597bc5 
cpu		=#a037b0 
wifi		=#71b9f8 
power		=#99ad6a
date		=#FFF
[bar/top]
dock = false
width = 100%
height = 27
offset-x = 0
offset-y = 0

underline-size = 2
underline-color = #00f

background=${colors.background}
foreground=${colors.foreground}
spacing = 1
padding-left = 0
padding-right = 2
module-margin-left = 1
module-margin-right = 2

font-0 = monaco:pixelsize=12;0
font-1 = Font Awesome:pixelsize=13;0

modules-left=i3
modules-center=xwindow
modules-right=backlight volume battery cpu memory wlan date 

tray-position = right
tray-padding = 2 

[module/xwindow]
type = internal/xwindow
label = %title%
label-maxlen = 30


[module/i3]
type = internal/i3
format = <label-state>
index-sort = true

label-focused = %index%
label-focused-background = #ee333333
label-focused-padding = 2

label-unfocused = %index%
label-unfocused-padding = 2

label-urgent = %index%!
label-urgent-background = #bd2c40
label-urgent-padding = 2

label-visible = %index%
label-visible-foreground = #55
label-visible-padding = 2

[module/backlight]
type=internal/xbacklight
label= %percentage%
format-underline=${colors.backlight}

[module/cpu]
type = internal/cpu
label =  %percentage%
format-underline =${colors.cpu} 

[module/memory]
type = internal/memory
label=   %percentage_used%
format-underline=${colors.memory}

[module/wlan]
type = internal/network
interface = wlp3s0
interval = 3.0

format-connected = <label-connected>

label-connected = %essid%
label-disconnected = %ifname%
label-disconnected-foreground = #55
format-disconnected-underline=${colors.wifi} 
format-connected-underline=${colors.wifi} 
[module/date]
type = internal/date
date =  %H:%M
date-alt =  %Y-%m-%d  %H:%M
interval = 5
format-underline=${colors.date} 

[module/volume]
type = internal/volume

format-volume =<ramp-volume> <label-volume>

label-volume =%percentage%

label-muted = MUTE
label-muted-foreground = #55

ramp-volume-0 =
ramp-volume-1 =
ramp-volume-2 =
format-volume-underline=${colors.volume}
format-muted-underline=${colors.volume}

[module/battery]
type = internal/battery
battery = BAT0
adapter = AC
full-at = 98

label-charging= %percentage%
format-charging = <label-charging>
format-discharging = <ramp-capacity> <label-discharging>
format-full =  <label-full>

ramp-capacity-0 =
ramp-capacity-1 =
ramp-capacity-2 = 
ramp-capacity-3 =
ramp-capacity-4 =

format-charging-underline=${colors.power} 
format-discharging-underline=${colors.power}
format-full-underline=${colors.power}
```

**Comment by jaagr** [comment]
What version are you running? `polybar -v`

**Comment by oliverrausch** [comment]
polybar 2.3.2 with i3wm on Antergos, the issue was resolved after upgrading to 2.3.5
